src/Organisms/DogData/DogData.js

```diff
@@ -2,17 +2,16 @@ import PropTypes from 'prop-types';
 import { useNavigate } from 'react-router-dom';
 
 import ColumnWrapper from '../../Templates/ColumnWrapper/ColumnWrapper';
-import DOGS from '../../Data/Dummy-data/test-data-dogs';
+import doggos from '../../Data/MongoDBMock/doggos';
 import DOG_DATA_TEMPLATE from '../../Consts/DogDataCONSTS';
 import DataLine from '../../Atoms/DataLine/DataLine';
 import SpecialButton from '../../Atoms/SpecialButton/SpecialButton';
 import SpecialButtonsContainerStyled from '../../Molecules/SpecialButtonsContainer/SpecialButtonsContainerStyled';
-import { doggoSex } from '../../Consts/formsDataToChose';
 import { getDataFormatDdMonthYyy } from '../../Tools/TimeFunctions';
 
 const DogData = ({ id }) => {
   let navigate = useNavigate();
-  const dog = DOGS.find((dog) => (dog.id = id));
+  const dog = doggos.find((dog) => (dog.id = id));
```

Comment:
In the future application this will be data to be fetched from ddb most probably, so my suggestion would be to already put it in the useEffect, where you will later use it. The schema for it could be:
1. create "dogState" - `const [dog, setDog] = useState()`
2. get the dog details and render component only if the data has came in correctly
`  useEffect(() => {
    async function fetchDogCetails(dogId) {
      //do async stuff
      const res = await fetch('doggourl')
      if (res.status !== 200) {
        // do stg
      } else {
        const dog = await res.json()
        if (!dog) {
          // do stg if no dog
        } else {
          setDog(dog)
        }
      }
    }
    fetchDogCetails()
  }, []);`

Reply:
Point 2 - for now - should be the place where you set the dog after finding it in the mongo db mock